lib/ansible/plugins/lookup/consul_kv.py

```diff
@@ -80,14 +82,17 @@ def __init__(self, loader=None, templar=None, **kwargs):
         self.agent_url = 'http://localhost:8500'
         if os.getenv('ANSIBLE_CONSUL_URL') is not None:
             self.agent_url = os.environ['ANSIBLE_CONSUL_URL']
+        self.agent_verify_certificate = True
+        if os.getenv('ANSIBLE_CONSUL_INSECURE_CERT') is not None:
+            self.agent_verify_certificate = False
```

Comment:
replace with
self.agent_verify_certificate = os.getenv('ANSIBLE_CONSUL_INSECURE_CERT') is None